.gitignore

```diff
@@ -1,5 +1,4 @@
 results
-data
```

Comment:
Keep this here -- it's so you can add data from the Snap datasets without interfering with git.
You can add your test dataset manually without removing the ignore here.